### ISSUE TITLE
How can I predict new data don't have labels

### ISSUE THREAD

**Issue by hongocvuong1998** [QingyongHu/RandLA-Net#102]
Hi @QingyongHu 
Thanks for your open source
I want to try your method on my data. When I using RandLaNet, I have a problem in preprocessing before fit to model training. As I see from source code, The author using labels for preprocessing. So, I don't know, When I predict new data don't have label, How can the model work?

**Comment by ytyfeng** [comment]
> Hi @QingyongHu

I have the same question. 
If I want to treat my unlabeled data as the test set and ignore training/validation split. What modifications do I need to do? 
I have already processed my data using `data_prepare_semantic3d.py`. 
In [line 48 of main_Semantic3D.py](https://github.com/QingyongHu/RandLA-Net/blob/master/main_Semantic3D.py#L48), it checks if label files exist and if not it only appends test data files. However, train and validation sets are used in `init_input_pipeline(self)`. It seems like the `flat_inputs` used to set up the network are based on [`batch_train_data`](https://github.com/QingyongHu/RandLA-Net/blob/master/main_Semantic3D.py#L329-L330).

**Comment by WuJing-coder** [comment]
Have you solved your problem? I just started studying. If it's convenient for you, I'd like to ask you for some help. Thank you.

**Comment by youssef962** [comment]
@ytyfeng @vvuonghn 
I'm facing the same problem and my data to be tested has no labels, have you found any solution?
